src/ingest_validation_tools/directory-schemas/mxif-v0.yaml

```diff
@@ -1,5 +1,31 @@
 files:
+  # Level0/
```

Comment:
A `dataset.json` file similar to CODEX is missing. This would essentially be a subset of CODEX's `dataset.json` because of the lack of compatibility between the two. This would have essential metadata like number of cycles, channel names, which channels are nuclear stains & membrane stains, etc. that HIVE tools use to process the data.